internal/mongodoc/package_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2015 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package mongodoc_test // import "gopkg.in/juju/charmstore.v5-unstable/internal/mongodoc"
```

Comment:
In this case the specifier doesn't match the package name, so almost certainly shouldn't be there. They would have been put there in the first place, by some over-eager automated script.